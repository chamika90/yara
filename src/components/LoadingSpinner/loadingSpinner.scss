@import "../../resources/colors.scss";

@keyframes spinner {
    0% {
      transform: rotate(0deg);
    }
    100% {
      transform: rotate(360deg);
    }
  }
  .loading-spinner {
    width: 20px;
    height: 20px;
    border: 5px solid $white; 
    border-top: 5px solid $black;
    border-radius: 50%;
    animation: spinner 1.5s linear infinite;
  }
  .spinner-container {
    display: flex;
    justify-content: center;
    align-items: center;
    text-align: center;
  }