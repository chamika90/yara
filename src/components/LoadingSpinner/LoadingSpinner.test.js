import { render } from "@testing-library/react";

import LoadingSpinner from "./LoadingSpinner";

describe('render loading spinner', () => {
    it('Should match snapshot', () => {
        const {toJSON}  = render(<LoadingSpinner />);
        expect(toJSON).toMatchSnapshot();
    });
});