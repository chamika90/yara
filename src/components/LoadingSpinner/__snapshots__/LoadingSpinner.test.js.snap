// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render loading spinner Should match snapshot 1`] = `undefined`;
